transport/tchannel/handler.go

```diff
@@ -132,6 +132,7 @@ func (h handler) callHandler(ctx context.Context, call inboundCall, responseWrit
 		Caller:          call.CallerName(),
 		Service:         call.ServiceName(),
 		Encoding:        transport.Encoding(call.Format()),
+		Transport:       "tchannel",
```

Comment:
Can we make these constants?

Reply:
These strings already existed as transportName in each transport package, so I used those.